test/units/plugins/action/test_synchronize.py

```diff
@@ -99,11 +99,11 @@ class SynchronizeTester(object):
     execute_called = False
 
 
-    def _execute_module(self, module_name, task_vars=None):
+    def _execute_module(self, module_name, module_args=None, task_vars=None):
```

Comment:
not sure if the test should modify the function signature ... we can end up with test code that would never work with the real thing ... which makes me question the tests.